packages/kbn-expandable-flyout/index.ts

```diff
@@ -16,4 +16,4 @@ export {
 export type { ExpandableFlyoutApi } from './src/context';
 
 export type { ExpandableFlyoutProps } from './src';
-export type { FlyoutPanel } from './src/types';
+export type { FlyoutPanelProps as FlyoutPanel } from './src/types';
```

Comment:
it seems that `FlyoutPanel` was only used in a few places outside of the `kbn-expandable-flyout` package. What do you think about renaming those places to use the new `FlyoutPanelProps` name? The PR would be slightly bigger but not by much and we would have consistency when people look at the package code and the apps code...

Reply:
yeah sorry ! :)